doc/benchmark/modules/ROOT/pages/preparing-keycloak.adoc

```diff
@@ -61,16 +61,24 @@ Some scenarios require a service account with the clientId `gatling`:
 
 Instead of following the above manual steps, you can use this link:{github-files}/benchmark/src/main/content/bin/initialize-benchmark-entities.sh[initialize-benchmark-entities.sh] script to do the setup for you.
 
+Specify the path to Keycloak distribution directory as KEYCLOAK_HOME.
+[source,shell]
+----
+export KEYCLOAK_HOME=provision/keycloak-cli/keycloak
+----
+
 Login to the Keycloak server using the kcadm cli script, which comes with any Keycloak distribution
 
 [source,shell]
 ----
-$KEYCLOAK_HOME/bin/kcadm.sh config credentials --server http://localhost:8081/auth --realm master --user admin --password admin
```

Comment:
Thanks for your suggestion @nehachopra27. If we are going to make the keycloak url to follow the pattern from minikube provisioning, we should refer the user to the installation guide or say something like `If you have installed the keycloak using minikube, we expect the url to look something like https://keycloak.192.168.12.345` and give a dummy ip in the example command